src/services/classifier.ts

```diff
@@ -771,6 +771,14 @@ namespace ts {
                 return false;
             }
 
+            // Limiting classification to exactly the elements and attributes
+            // defined in `ts.commentPragmas` would be excessive, but we can avoid
+            // some obvious false positives (e.g. in XML-like doc comments) by
+            // checking the element name.
+            if (!match[3] || !(commentPragmas as any)[match[3]]) {

```

Comment:
I believe you can write `!(match[3] in commentPragmas)` without a cast?

Reply:
Didn't know about that operator.  Updating.